doc/2/guides/develop-on-kuzzle/1-embedded-sdk/index.md

```diff
@@ -0,0 +1,161 @@
+
+You can access it through the [Backend.sdk](/core/2/some-link) property. 
+
+## Controllers
+
+The following controllers are available in the embedded SDK:
+
+- [auth](/sdk/js/7/controllers/auth)
+- [bulk](/sdk/js/7/controllers/bulk)
+- [collection](/sdk/js/7/controllers/collection)
+- [document](/sdk/js/7/controllers/document)
+- [index](/sdk/js/7/controllers/index)
+- [ms (memoryStorage)](/sdk/js/7/controllers/ms)
+- [security](/sdk/js/7/controllers/security)
+- [server](/sdk/js/7/controllers/server)
+- [realtime](/sdk/js/7/controllers/realtime)
+
+::: warning
+The behavior of the [realtime:subscribe](/sdk/js/7/controllers/realtime) method is slightly different when it's used with the Embedded SDK.  
+Learn more about [Backend Realtime Subscriptions](/core/2/guides/develop-on-kuzzle/1-embedded-sdk#backend-realtime-subscriptions)
+:::
+
+**Example:** _Create a new document by using the [document.create](/sdk/js/7/controllers/document/create) method_
+```js
+// After application startup
+
+// [...]
+
+// Creates a document
+await app.sdk.document.create('nyc-open-data', 'yellow-taxi', {
+  name: 'Aschen',
+  age: 27
+})
+```
+
+## Query method
+
+<!-- Duplicate /core/2/guides/getting-started/6-write-application -->
+
+The low level [query](/sdk/js/7/core-classes/kuzzle/query) method can also be used to **send custom requests to the Kuzzle API**.  
+
+**Example:** _Execute a custom controller action with the [query](/sdk/js/7/core-classes/kuzzle/query) method_
+```js
+// After application startup
+
+// [...]
+
+// Execute a custom controller action
+await app.sdk.query({
+  controller: 'greeting',
+  action: 'name',
+  name: 'Aschen'
+})
+```
+
+## User impersonation
+
+By default, when using the Embedded SDK, requests **don't have the same context as the original request** received by Kuzzle.
+
+Typically, the `request.context.user` property is not set and thus **[Kuzzle metadata](/core/2/some-link) will not be set when creating or updating documents**.
+
+It is possible to use the same user context as the original request with the Embedded SDK, for this purpose it is necessary to use the [EmbeddedSDK.as](/core/2/some-link) method.
+
+**Example:** _Creating a document as the original API user to preserv Kuzzle metadata_
+```js
+app.controller.register('drivers', {
+  actions: {
+    create: {
+      handler: async request => {
+        const originalUser = request.context.user
+
+        return app.sdk.as(originalUser).document.create(
+          'nyc-open-data',
+          'yellow-taxi',
+          { name: 'Aschen' })
+       }
+    }
+  }
+})
+```
+
+::: warning
+User permissions are not applied even when the [EmbeddedSDK.as](/core/2/some-link) method is used.
+:::
+
+## Backend Realtime Subscriptions
+
+Realtime subscriptions should be made using the [Realtime Controller](/sdk/js/7/controllers/realtime) **just after the application startup**.
```

Comment:
An explanation of the differences with external calls to `realtime:subscribe` is in order here, I felt myself a bit lost with this feature (I'm still not accustomed to it :grin:).

This explanation also helps when introducting the `propagate` option below, as it gives a bit more context to the reader.

```suggestion
Realtime subscriptions should be made using the [Realtime Controller](/sdk/js/7/controllers/realtime) **just after the application has started**.  

Realtime subscriptions performed by an application are used so that an application gets notified about changes, and can act upon them. The behavior is much the same than when a client subscribes, but since the entity performing the subscription is different (client vs. application), the feature accessible by an application has some new options to fine tune how notifications are propagated across a Kuzzle cluster.
```

Reply:
Thanks :D